apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -54,10 +55,16 @@
     when Key :: dcos_l4lb_mesos_poller:key(),
          Backend :: dcos_l4lb_mesos_poller:backend()).
 push_vips(VIPs) ->
+    Begin = erlang:monotonic_time(),
     try
         gen_server:call(?MODULE, {vips, VIPs})
     catch exit:{noproc, _MFA} ->
         ok
+    after
+        prometheus_summary:observe(
```

Comment:
Could you please help me understand why in some places calls to `observe` are placed into `after` branches, and in other places they are not?

Reply:
In this case, it is because we want to count the time, regardless of the call failing or not. I am unsure what happens if the pushing of VIPs fails partially, so I also update the number of VIPs.

I would usually they are not on the `after` block unless we have a reason to put them there. 

What do you think about this perspective?

Reply:
ditto;